authorization-tck/pom.xml

```
@@ -32,6 +32,9 @@
     <name>TCK: Authorization</name>
 
     <properties>
+        <tck.root>${project.build.directory}</tck.root>
+        <maven.settings.xml>${user.home}/.m2/settings.xml</maven.settings.xml>
+        <maven.executable>${env.M2_HOME}/bin/mvn</maven.executable>
```

M2_HOME is the old variable name, we should use the new one
```suggestion
        <maven.executable>${env.MAVEN_HOME}/bin/mvn</maven.executable>
```

---

It would be great if this was also automatically adjusted for Windows by a profile something along the lines of:

```
<profile>
    <id>windows</id>
    <activation>
        <os>
            <family>Windows</family>
        </os>
    </activation>
    <properties>
        <maven.executable>${env.MAVEN_HOME}${file.separator}bin${file.separator}mvn.cmd</maven.executable>
    </properties>
</profile>
```